website/pages/commands/connect/envoy.mdx

```diff
@@ -75,7 +75,7 @@ proxy configuration needed.
   allowed to access by [Connect intentions](/docs/connect/intentions).
 
 - `-envoy-version` - The version of envoy that is being started. Default is
-  `1.14.4`. This is required so that the correct configuration can be generated.
+  `1.14.5`. This is required so that the correct configuration can be generated.
```

Comment:
I edited these website refs because it was simple, but it's dead code since we don't render these website chunks, nor keep them up to date.